src/content/guides/creating-clusters-via-crs-on-aws/index.md

```diff
@@ -0,0 +1,175 @@
+
+```
+apiVersion: infrastructure.giantswarm.io/v1alpha2
+kind: AWSMachineDeployment
+metadata:
+  labels:
+    aws-operator.giantswarm.io/version: 8.1.1
+    giantswarm.io/cluster: nzr5z
+    giantswarm.io/machine-deployment: pv7ps
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.0.1
+  name: pv7ps
+  namespace: default
+spec:
+  nodePool:
+    description: np1
+    machine:
+      dockerVolumeSizeGB: 100
+      kubeletVolumeSizeGB: 100
+    scaling:
+      max: 10
+      min: 3
+  provider:
+    availabilityZones:
+    - eu-central-1a
+    worker:
+      instanceType: m4.xlarge
+```
+
+So, when the release 10.x.x+ in AWS installation is used, *Giant Swarm API* service creates *Cluster* and *AWSCluster* CRs in Control-Plane Kubernetes.
+
+
+## MachineDeployment CRs
+
+The same approach applies to node pools management, where high level *MachineDeployment* CR represents Cluster API generic node pool implementation. 
+
+*MachineDeployment* CR object example:
+
+```
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: MachineDeployment
+metadata:
+  labels:
+    cluster-operator.giantswarm.io/version: 2.1.1
+    giantswarm.io/cluster: nzr5z
+    giantswarm.io/machine-deployment: pv7ps
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.0.1
+  name: pv7ps
+  namespace: default
+spec:
+  selector: {}
+  template:
+    metadata: {}
+    spec:
+      bootstrap: {}
+      infrastructureRef:
+        apiVersion: infrastructure.giantswarm.io/v1alpha2
+        kind: AWSMachineDeployment
+        name: pv7ps
+        namespace: default
+      metadata: {}
+```
+
+As with *Cluster* CR, *MachineDeployment* has the reference to infrastructure object. In this case, it is *AWSMachineDeployment*, which represents the Giant Swarm Cluster API implementation of node pool for the AWS platform.
+
+*AWSMachineDeployment* CR object example:
+
+```
+apiVersion: infrastructure.giantswarm.io/v1alpha2
+kind: AWSMachineDeployment
+metadata:
+  labels:
+    aws-operator.giantswarm.io/version: 8.1.1
+    giantswarm.io/cluster: nzr5z
+    giantswarm.io/machine-deployment: pv7ps
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.0.1
+  name: pv7ps
+  namespace: default
+spec:
+  nodePool:
+    description: np1
+    machine:
+      dockerVolumeSizeGB: 100
+      kubeletVolumeSizeGB: 100
+    scaling:
+      max: 10
+      min: 3
+  provider:
+    availabilityZones:
+    - eu-central-1a
+    worker:
+      instanceType: m4.xlarge
+```
+
+## How to create the cluster omitting Giant Swarm API?
+
+All the CRs, mentioned above, have the strict spec and important requirements to be considered valid. 
+There is no cluster/node pool CRs validation available in Control Plane Kubernetes of Giant Swarm for now. 
+Therefore, if you create CR with wrong field values, that can result in a broken tenant cluster, which then causes unwanted alerts in Giant Swarm monitoring system.
```

Comment:
I'm not sure if users care about unwanted alerts, just having broken clusters is bad enough :D